File: back-csharp/_dtos/UniversityDto.cs
using System.Diagnostics.CodeAnalysis;
using System.Text.Json.Serialization;

namespace back_csharp._dtos;

public class UniversityDto
{
    [JsonPropertyName("university_id")]
    public int UniversityId { get; set; }
    [JsonPropertyName("name")]
    public string Name { get; set; }
    [JsonPropertyName("img_file")]
    public string? ImgFile { get; set; }
    [JsonPropertyName("img_type")]
    public string? ImgType { get; set; }
}

File: back-rs/src/dtos/university_dto.rs
use serde::{Deserialize, Serialize};

#[derive(Serialize, Deserialize)]
#[derive(Debug)]
pub struct UniversityDTO {
    pub university_id : Option<i32>,
    pub name : String,
    pub img_file : Option<String>,
    pub img_type : Option<String>
}